### ISSUE TITLE
Bidder Targeting Key status is 'INACTIVE'

### ISSUE THREAD

**Issue by dshore** [prebid/line-item-manager#23]
* line-item-manager version: all
* Python version: all
* Operating System: all

### Description

Create line items when bidder targeting key status is 'inactive' results in an error.

### What I Did

```
line_item_manager create <config.yml> -b <bidder-code>
```

**Comment by dshore** [comment]
Now doing a check of all bidders up front making sure targeting keys are 'ACTIVE'.  Errors look like this:

```
$ line_item_manager create my_config.yml -b appnexus -n
ERROR:line_item_manager.cli:Resource is not active:
  - Bidder Targeting Key name 'hb_pb_appnexus' is not active.  You must activate before re-running.
```

**Comment by dshore** [comment]
Released with: v0.1.15
